packages/@aws-cdk/aws-rds/lib/cluster-ref.ts

```diff
@@ -46,38 +46,50 @@ export interface IDatabaseCluster extends IResource, ec2.IConnectable, secretsma
  */
 export interface DatabaseClusterAttributes {
   /**
-   * The database port
+   * Identifier for the cluster
    */
-  readonly port: number;
+  readonly clusterIdentifier: string;
```

Comment:
any reason we wouldn't want to just call it `clusterId` - don't think we expand it out to `identifier` in many places do we?